Examples/Blink/Blink.ioc

```diff
@@ -0,0 +1,178 @@
+ProjectManager.TargetToolchain=Makefile
+SH.GPXTI13.ConfNb=1
+Mcu.ThirdPartyNb=0
+PA3.GPIO_Mode=GPIO_MODE_AF_PP
+RCC.HCLKFreq_Value=72000000
+Mcu.IPNb=4
+RCC.I2SClocksFreq_Value=72000000
+ProjectManager.PreviousToolchain=
+RCC.APB2TimFreq_Value=72000000
+Mcu.Pin6=PA3
+Mcu.Pin7=PA5
+ProjectManager.RegisterCallBack=
+Mcu.Pin8=PA13
+RCC.USBFreq_Value=72000000
+Mcu.Pin9=PA14
+PC15-OSC32_OUT.Signal=RCC_OSC32_OUT
+RCC.AHBFreq_Value=72000000
+RCC.TIM2Freq_Value=72000000
+Mcu.Pin0=PC13
+Mcu.Pin1=PC14-OSC32_IN
+PF0-OSC_IN.Mode=HSE-External-Clock-Source
+Mcu.Pin2=PC15-OSC32_OUT
+Mcu.Pin3=PF0-OSC_IN
+RCC.USART3Freq_Value=36000000
+Mcu.Pin4=PF1-OSC_OUT
+Mcu.Pin5=PA2
+PA5.Signal=GPIO_Output
+ProjectManager.ProjectBuild=false
+RCC.HSE_VALUE=8000000
+NVIC.UsageFault_IRQn=true\:0\:0\:false\:false\:true\:true\:false
+NVIC.DebugMonitor_IRQn=true\:0\:0\:false\:false\:true\:true\:false
+board=NUCLEO-F303RE
+USART2.VirtualMode-Asynchronous=VM_ASYNC
+NVIC.SysTick_IRQn=true\:0\:0\:true\:false\:true\:true\:true
+RCC.RCC_PLLsource_Clock_Source_FROM_HSE=RCC_HSE_PREDIV_DIV2
+ProjectManager.LastFirmware=true
+RCC.PLLMUL=RCC_PLL_MUL9
+RCC.VCOOutput2Freq_Value=8000000
+ProjectManager.FirmwarePackage=STM32Cube FW_F3 V1.11.1
+MxDb.Version=DB.6.0.10
+RCC.APB2Freq_Value=72000000
+RCC.TIM8Freq_Value=72000000
+RCC.UART4Freq_Value=36000000
+ProjectManager.BackupPrevious=false
+MxCube.Version=6.1.0
+PC14-OSC32_IN.Mode=LSE-External-Oscillator
+RCC.I2C1Freq_Value=8000000
+PA14.Mode=Serial_Wire
+PF0-OSC_IN.Locked=true
+PC13.GPIOParameters=GPIO_Label,GPIO_ModeDefaultEXTI
+PA5.GPIO_PuPd=GPIO_NOPULL
+File.Version=6
+VP_SYS_VS_Systick.Mode=SysTick
+PB3.GPIOParameters=GPIO_Label
+NVIC.NonMaskableInt_IRQn=true\:0\:0\:false\:false\:true\:true\:false
+NVIC.PendSV_IRQn=true\:0\:0\:false\:false\:true\:false\:false
+RCC.UART5Freq_Value=36000000
+PA13.Mode=Serial_Wire
+ProjectManager.FreePins=false
+RCC.IPParameters=ADC12outputFreq_Value,ADC34outputFreq_Value,AHBFreq_Value,APB1CLKDivider,APB1Freq_Value,APB1TimFreq_Value,APB2Freq_Value,APB2TimFreq_Value,CortexFreq_Value,FCLKCortexFreq_Value,FamilyName,HCLKFreq_Value,HSE_VALUE,HSI_VALUE,I2C1Freq_Value,I2C2Freq_Value,I2C3Freq_Value,I2SClocksFreq_Value,LSI_VALUE,MCOFreq_Value,PLLCLKFreq_Value,PLLM,PLLMCOFreq_Value,PLLMUL,PLLN,PLLP,PLLQ,RCC_PLLsource_Clock_Source_FROM_HSE,RTCFreq_Value,RTCHSEDivFreq_Value,SYSCLKFreq_VALUE,SYSCLKSourceVirtual,TIM15Freq_Value,TIM16Freq_Value,TIM17Freq_Value,TIM1Freq_Value,TIM20Freq_Value,TIM2Freq_Value,TIM3Freq_Value,TIM8Freq_Value,UART4Freq_Value,UART5Freq_Value,USART1Freq_Value,USART2Freq_Value,USART3Freq_Value,USBFreq_Value,VCOOutput2Freq_Value
+ProjectManager.AskForMigrate=true
+Mcu.Name=STM32F303R(D-E)Tx
+ProjectManager.HalAssertFull=false
+RCC.RTCHSEDivFreq_Value=250000
+PA2.Signal=USART2_TX
+PA13.GPIO_Label=TMS
+ProjectManager.ProjectName=Blink
+ProjectManager.UnderRoot=false
+RCC.TIM1Freq_Value=72000000
+PA13.GPIOParameters=GPIO_Label
+RCC.PLLMCOFreq_Value=72000000
+ProjectManager.CoupleFile=false
```

Comment:
The "generate peripheral initialization as a pai of '.c/.h' files per peripheral" option must be set as the generated code will go to main.c, which is being deleted

Reply:
Good point, fixed